frb_codegen/src/lib.rs

```diff
@@ -77,3 +78,13 @@ pub fn frb_codegen_multi(
     info!("Success!");
     Ok(())
 }
+
+pub fn frb_codegen_common_files(configs: &[Opts]) -> anyhow::Result<()> {
```

Comment:
nit: shall this be a `pub` fn (i.e. exposed as public API)?

Reply:
It should be accessible from `main.rs`, so I set it as public. 

The related function should find common structs by looking at all `IrFile` at once, not each one individually. That's why I thought it should be called in `main.rs`.

Reply:
Ok I see. Hopefully a later refactor can unify many things :)